src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1666,6 +1679,15 @@ private bool AddOrganelle(string organelleType)
         if (placementActions.Count < 1)
             return false;
 
+        if (organelleType == "flagellum")
```

Comment:
This should be implemented by getting the OrganelleDefinition in _Ready by the name and comparing the name from the flagellum organelle definition here. This way it ensures if the organelle internal name is renamed we can easily detect the bug by the editor crashing instead of this feature silently stopping working.